cmd/shim/main.go

```diff
@@ -44,9 +44,10 @@ const (
 )
 
 var (
-	stdoutPath = metaMountPath + "/stdout"
-	stderrPath = metaMountPath + "/stderr"
-	pipeWg     sync.WaitGroup
+	stdoutPath   = metaMountPath + "/stdout"
+	stderrPath   = metaMountPath + "/stderr"
+	combinedPath = metaMountPath + "/combined"
```

Comment:
Any chance we could also expose this through the graphql api? Maybe named something like `Output`? :tada:

Not immediately required for this PR though!

Reply:
Unfortunately I'm gonna back this change out instead to manage scope creep, since it includes changes to SDKs + docs and is technically a backwards incompatible change. :(